ads/inabox/inabox-messaging-host.js

```diff
@@ -80,20 +80,22 @@ export class InaboxMessagingHost {
    * @param {!Array<!HTMLIFrameElement>} iframes
    */
   constructor(win, iframes) {
+    const hostWin = canInspectWindow(win.top) ? win.top : win;
```

Comment:
any reason why do we prefer top window?

Reply:
We're passing the window to the position observer below, which would measure elements' positions relative to it. As a result we want it to be the top window if possible.

Reply:
got you, please add a comment to explain

Reply:
Done.